gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+// Reveal problems with box-sizing set to border-box, see https://issues.sonatype.org/browse/RSC-84
```

Comment:
Is this comment still valid?

Reply:
> Is this comment still valid?

No: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/f4fbce248894a3a44d33b33bb3ed5a4ed0d8dac3